src/components/FormLayout/FormLayout.scss

```diff
@@ -1,6 +1,12 @@
+/* stylelint-disable function-name-case */
 @import '../../styles/common';
 
-$item-min-size: rem(220px);
+$item-min-size: rem(200px);
+$container-width: rem(684px);
+// This is a trick for container aware responsive design
+// Inspired from: https://heydonworks.com/article/the-flexbox-holy-albatross/
+$modifier: calc(#{$container-width} - 100%);
+$modifier-value: calc(#{$modifier} * 999);
```

Comment:
As outlined by the inline comments, this is a homebrew version of the `@container` queries. Not sure if there was any alternatives to this in our toolbox, and not sure if it's too hacky. 

Meanwhile, it is tested in BrowserStack using Chrome, Firefox, Safari, Opera, iOS Safari + Chrome, and Android Chrome, and it all work as expected.